arm-machinelearning/2016-05-01-preview/swagger/webservices.json

```diff
@@ -683,6 +670,32 @@
         "type"
       ]
     },
+    "ExampleRequest": {
+      "type": "object",
+      "description": "Sample input data for the service's input(s).",
+      "properties": {
+        "inputs": {
+          "type": "object",
+          "description": "Sample input data for the web service's input(s) given as an input name to sample input values matrix map.",
+          "additionalProperties": {
```

Comment:
Oh I see, are you trying to say that this is a dictionary of string, array of array of objects?
```
{
  'string': [
    [{}, {}, {}], 
    [{}, {}]
  ]
}
```